vuu-ui/packages/vuu-layout/src/flexbox/FlexboxLayout.tsx

```diff
@@ -19,7 +23,7 @@ export const FlexboxLayout = function FlexboxLayout(props) {
     [dispatch, path]
   );
 
-  return <Flexbox {...props} onSplitterMoved={handleSplitterMoved} />;
+  return <Flexbox {...props} onSplitterMoved={handleSplitterMoved} ><></><></></Flexbox>;
```

Comment:
what does this change do? :)

Reply:
It resolves an error (badly) and then I forgot about it, probably easiest to revert and leave as is for now - I think this will be rewritten later down the line when we know what we want to do with it